spec/unit/inspection_spec.rb

```diff
@@ -102,9 +102,13 @@
 end
 
 describe 'aasm.states_for_select' do
-  it "should return a select friendly array of states" do
-    expect(Foo.aasm).to respond_to(:states_for_select)
-    expect(Foo.aasm.states_for_select).to eq([['Open', 'open'], ['Closed', 'closed'], ['Final', 'final']])
+  context 'without I18n' do
```

Comment:
After requiring `I18n` by default in specs ([this change](https://github.com/aasm/aasm/pull/689/files#diff-93830fa29d616f7c87903d08b5b1b29aR16)), some specs start failing because of this condition https://github.com/aasm/aasm/blob/c270f656b9a3b0dcd299234ef18ecadc01630387/lib/aasm/core/state.rb#L58-L62

... So I made a workaround in such specs.